README.md

```diff
@@ -1,37 +1,71 @@
-Bowling Challenge in Ruby
-=================
+# Bowling Challenge in Ruby
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+This is Alexis Morcrette's partial solution to the Bowling Challenge in Ruby, Makers Academy's Week 5 Weekend Challenge.
 
-## The Task
+The focus for this challenge is to write high-quality code, to deliver a bowling score program.
 
-**THIS IS NOT A BOWLING GAME, IT IS A BOWLING SCORECARD PROGRAM. DO NOT GENERATE RANDOM ROLLS. THE USER INPUTS THE ROLLS.**
+This project demonstrates the following skills:
+* Using diagramming to plan my approach to the challenge
+* TDD my code
+* Focus on testing behaviour rather than state
+* Committing often, with good commit messages
+* Single Responsibility Principle and encapsulation
+* Clear and readable code
+
```

Comment:
README has clear instructions. Skills demonstrated is listed very nicely and gives the reader a good idea of the process of creating the project.